static/search.js

```diff
@@ -320,6 +320,7 @@ var Search = Control.extend({
 		if (index && currentIndexVersion === indexVersion) {
 			searchEngine = lunr.Index.load(index);
 		}else{
+			var dummyContainer = document.createElement('div');
```

Comment:
I merged this as-is and worked around it by running this code before the items are sent to the web worker.